### DISCUSSION TITLE
How to managing multiple assemblys of OpenSlides

### DISCUSSION THREAD

**Discussion by InterestingIndeed** [OpenSlides/OpenSlides#5968] [Q&A]
I'm about to test the installation of OpenSlides 4 on a VPS in about 2 weeks. My goal is to have multiple assemblies organized (not run) at the same time. Is that possible with a single installation of OpenSlides or do I have to install multiple versions and configure different URLs to connect to those versions with specific ports?

e.g.
www.assembly1.com -> IP:9999 -> openslides setup 1
www.assembly2.com -> IP:8888 -> openslides setup 2

**Comment by FinnStutzenstein** [comment]
With OpenSlides 4 it is possible, but the development state does not have this feature yet. In OS3 you can start the setup two times. You have to adjust your ports for each setup.

**Reply by InterestingIndeed** [reply]
Okay, great, thank you very much!
